release-notes/release-notes52.md

```diff
@@ -125,10 +146,37 @@ The {{site.productname}} 5.2 release includes **Spellchecker Pro** 2.0.
 
 {{site.productname}} 5.2 provides fixes for the following bugs:
 
--
+- Fixed the incorrect parsing of malformed/bogus HTML comments.
+- Fixed the `quickbars` selection toolbar appearing on non-editable elements.
+- Fixed a bug with alignment toolbar buttons sometimes not changing state correctly.
+- Fixed the `codesample` toolbar button not toggling when selecting code samples other than HTML.
+- Fixed content incorrectly scrolling to the top or bottom when pressing enter when the content was already in view.
+- Fixed the `scrollIntoView` event potentially hiding elements behind the toolbar.
+- Fixed the editor not respecting the `resize_img_proportional` setting due to legacy code.
+- Fixed the floating toolbar drawer flickering in inline mode.
+- Fixed an issue where the `template` plugin dialog would be indefinitely blocked on a failed template load.
+- Fixed the `mscontrolselect` event not being unbound on the Microsoft Internet Explorer and Microsoft Edge browsers.
+- Fixed the "Confirm" dialog footer buttons to only highlight the "Yes" button.
+- Fixed the `file_picker_callback` functionality for the Image, Link and Media plugins.
+- Fixed an issue where the floating toolbar drawer would sometimes break if the editor is resized while the drawer is open.
+- Fixed an incorrect `external_plugins` loading error message.
+- Fixed the resize handler not being hidden for ARIA purposes. Patch contributed by [Parent5446](https://github.com/Parent5446).
+- Fixed an issue where content could be lost if a misspelled word was selected and spellchecking was disabled.
+- Fixed validation errors in the CSS where certain properties had the wrong default value.
+- Fixed an issue where forced root block attributes were not applied when removing a list.
+- Fixed an issue where the element path was not cleared when there are no parents.
+- Fixed an issue where width and height in svg icons containing `rect` elements were overridden by the CSS reset.
+- Fixed an issue where uploading images with `images_reuse_filename` enabled and that included a query parameter would generate an invalid URL.
```

Comment:
"enabled and that" <- something feels odd

Reply:
- Fixed an issue where uploading images with a query parameter and the `images_reuse_filename` setting enabled would generate an invalid URL.